packages/omniapi/src/public/Datasets_Branch.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type {
+  ClientContext as $ClientContext,
+  OmniMethod as $OmniMethod,
+} from "@osdk/shared.net";
+import { omniFetch as $omniFetch } from "@osdk/shared.net";
+import type * as $C from "../generated/components.js";
+
+const _listBranches: $OmniMethod<
```

Comment:
These are all generated right? (everything in public)

Reply:
yeah